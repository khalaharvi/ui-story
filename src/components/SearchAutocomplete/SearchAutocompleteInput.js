import React from 'react';

import { Search as SearchIcon } from 'icons';
import { cx } from './SearchAutocomplete';

const SearchAutocompleteInput = props => (
  <div className={cx('input-container')}>
    <input {...props} type="search" />
    <button className={cx('input-button')} type="submit">
      <SearchIcon className={cx('input-icon')} />
    </button>
  </div>
);

export default SearchAutocompleteInput;
