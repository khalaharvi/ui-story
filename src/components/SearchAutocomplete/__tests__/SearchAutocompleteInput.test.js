import React from 'react';
import { shallow } from 'enzyme';

import SearchAutocompleteInput from '../SearchAutocompleteInput';

describe('<SearchAutocompleteInput />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SearchAutocompleteInput />);
    expect(wrapper).toMatchSnapshot();
  });
});
