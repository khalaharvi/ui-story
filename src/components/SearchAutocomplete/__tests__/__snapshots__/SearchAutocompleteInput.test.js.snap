// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchAutocompleteInput /> renders correctly 1`] = `
<div
  className="dh-search-autocomplete-input-container"
>
  <input
    type="search"
  />
  <button
    className="dh-search-autocomplete-input-button"
    type="submit"
  >
    <Search
      className="dh-search-autocomplete-input-icon"
    />
  </button>
</div>
`;
